config/project.cmake

```diff
@@ -38,11 +38,12 @@ else()
   message(STATUS "Note: using 32 bit integer ids.")
 endif()
 
+cinch_load_extras()
 #------------------------------------------------------------------------------#
 # Add library targets
 #------------------------------------------------------------------------------#
 
-cinch_add_library_target(flecsi-sp specializations)
+cinch_add_library_target(flecsi-sp flecsi-sp)
```

Comment:
All "#include" statement of flecsi-sp uses "flecsi-sp" as a parent-dir, not "specializations". It seems like that was due to old-version of cinch created symbolic link to the header inside build directory but created the directory such as "build/include/flecsi-sp" that is linked to "flecsi-sp/specializations". This symlink directory no longer exists during build with newer cinch. 

The easiest way to mitigate this non-backward compatibility of the cinch is to simply change the directory structure of flecsi-sp to similar to other packages that use newer cinch, such as flecsi and flecsale. These packages appears to have a directory structure like "flecsi/flecsi" and "flecsale/flecsale". 

If this is a problem, then please guide me through how I should fix them.